app/controllers/providers_controller.rb

```diff
@@ -235,23 +235,23 @@ def stats
       providers = provider_count(consortium_id: nil)
       clients = client_count(provider_id: nil)
       dois = doi_count(provider_id: nil)
-      resource_types = resource_type_count(provider_id: nil)
+      # resource_types = resource_type_count(provider_id: nil)
```

Comment:
In the next step, this could be integrated with doi_count into a single elasticsearch query and two aggregations.